test/mirage/action/test.ml

```diff
@@ -14,7 +14,8 @@ let print_banner s =
   print_newline ()
 
 let info context =
-  Info.v ~packages:[] ~keys:[] ~runtime_args:[] ~build_cmd:"mirage build"
+  Info.v ~packages:[] ~keys:[] ~runtime_args:[]
+    ~build_cmd:(fun _ -> "make build")
```

Comment:
given that we discard the path argument here, do we really need to change the `build` above to be a `Fpath.t -> string` function?

Reply:
Yes, we do. This is the `test/mirage` directory. In the "normal" generated opam file, the subdirectory is not discarded.

In e.g. the `caldav` repository (https://github.com/robur-coop/caldav), if you `cd mirage && mirage configure -t hvt`, the resulting opam file will now include: `build: [ make "-Cmirage" "build" ]`.

I added aff5e9f4 which uses the argument provided also in the test.